.github/workflows/DeployPreview.yml

```
@@ -27,35 +27,36 @@ jobs:
     runs-on: ubuntu-latest
     defaults:
       run:
-        working-directory: ./playground
+        working-directory: ./website
     steps:
       - name: Checkout
         uses: actions/checkout@v3
-      - name: Setup output directory
-        run: mkdir build
       - name: Install toolchain (minimal, stable, wasm32-unknown-unknown + wasm32-wasi)
         uses: dtolnay/rust-toolchain@stable
         with:
           targets: wasm32-unknown-unknown, wasm32-wasi
       - name: Install wasm-pack
         uses: jetli/wasm-pack-action@v0.4.0
-      - name: Make build script executable
-        run: chmod +x ./build-playground.sh
-      - name: Build Playground
-        run: ./build-playground.sh
+      - name: Build website
+        run: |
+          npm ci
+          chmod +x ./build_website.sh
+          export BASE="/pr-preview/pr-${PR}" && ./build_website.sh
+        env:
+          PR: ${{ github.event.number }}
```

You can just set `BASE` directly:
```suggestion
          BASE: /pr-preview/pr-${{ github.event.number }}
```
Would look cleaner